sections/exchange/TradeCard/CurrencyCard/CurrencyCard.tsx

```diff
@@ -246,27 +322,44 @@ const Slippage = styled.div`
 	color: ${(props) => props.theme.colors.yellow};
 `;
 
-const WalletBalanceContainer = styled(FlexDivRowCentered)<{ disableInput?: boolean }>`
+const CurrencyNameLabel = styled.div`
+	text-transform: capitalize;
+	text-align: left;
+	font-size: 14px;
+	font-family: ${(props) => props.theme.fonts.regular};
+	line-height: 1.25em;
+	color: ${(props) => props.theme.colors.common.secondaryGray};
+	width: 161px;
+	padding-left: 12px;
+`;
+
+const WalletBalanceContainer = styled(FlexDivRow)<{ disableInput?: boolean }>`
 	${(props) =>
 		props.disableInput &&
 		css`
 			pointer-events: none;
 		`}
+	width: 136px;
```

Comment:
```suggestion
	width: 100%;
```

Reply:
Increased the width of  `StyledBaseModal` to 300px.